### DISCUSSION TITLE
articles/membuat-component-progress-bar-nextjs

### DISCUSSION THREAD

**Discussion by giscus** [reggi49/comments#3] [kodepintar] [answered]
# articles/membuat-component-progress-bar-nextjs

Get the latest news about Programming, Technology, Gadgets, Games, Software and Hardware. Explore your creativity with the Internet of Things and Cloud Computing

https://kodeabc.xyz/articles/membuat-component-progress-bar-nextjs

**Comment by reggi49** [comment, marked as answer]
if any questions, please leave comments
